### ISSUE TITLE
Keyboard shortcuts open find bar with "Search for text when you start typing"

### ISSUE THREAD

**Issue by kewisch** [mozilla/addons-code-manager#770]
### Describe the problem and steps to reproduce it:

* In Firefox preferences, enable search for text when you start typing 
* In the code manager, use one of the keyboard shortcuts

### What happened?

* The shortcut works, but the find bar also opens and displays the shortcut.

### What did you expect to happen?

* The find bar doesn't open

### Anything else we should know?

Likely missing an event.preventDefault() somewhere.

**Comment by AlexandraMoga** [comment]
Verified on -dev with Fx67 Win10x64

The shortcut keys are now helping navigation inside the file tree/file viewer instead of triggering the quick find bar
As a small drawback, if you are trying to search for a term that starts with one of the predefined navigation keys, the first one or two letters will most likely not be populated. 


- in the following screencats, my first actions are `c` to close the file tree, then `o` to open it, `j` to navigate to the next files then `k` to go to the previous files. Finally, I'm entering a search term to see if thequick find bar is triggered. 
![2019-06-04_16-00-54](https://user-images.githubusercontent.com/31961530/58881402-cfe2f800-86e2-11e9-90b5-a57f305d3c9a.gif)
